### ISSUE TITLE
The learning rate of Dream

### ISSUE THREAD

**Issue by lll2343** [DreamLM/Dream#29]
Great work, is it possible to know the learning rate of training dream from qwen

**Comment by jiacheng-ye** [comment]
We have recently released our fine-tuning script. Check it out!
